### PR TITLE
added additional logging to ftl for benchmarking

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: ftl/common/builder.py [hunk 1]
```diff
@@ -73,17 +73,21 @@ def CreatePackageBase(self, base_image, cache):
...
                 info = tarfile.TarInfo(os.path.join('app', name))
                 info.size = len(content)
                 out.addfile(info, fileobj=cStringIO.StringIO(content))
+        logging.info('Finished generating tarfile from context.')
 
         tar = buf.getvalue()
         sha = 'sha256:' + hashlib.sha256(tar).hexdigest()
 
         gz = cStringIO.StringIO()
+        logging.info('Starting to gzip tarfile...')
```
Comment (dlorenc) [inline ftl/common/builder.py]: Do these include timestamps by default when printed, or do we need to customize the formatter?

Comment (aaron-prindle) [reply ftl/common/builder.py]: no it didn't.  I just updated the logging configuration to have millisecond timings so now it does.

**Comment by dlorenc** [comment]
Looks like there's a flake issue with line length.

**Comment by aaron-prindle** [comment]
passing now

**Pull request merged**